pegasus/sites.v3/code.org/public/index.haml

```diff
@@ -52,8 +52,10 @@ social:
 - # The homepage gsheet may specify "en" or "not en" for many row types.
 - languageFilter = request.language == "en" ? "en" : "not en"
 
+- page_mode = PageMode.get(request)
```

Comment:
I removed it from the pages where we only needed it before we flipped the switch. I also had to move it the homepage top level because when it was called in homepage_hero and homepage_sections they could be different. Then you would get the same tutorial in both the hero and hoc block.